src/validate/test.rs

```diff
@@ -0,0 +1,117 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+
+use validate;
+
+#[test]
+fn aws_region()
+{
+  assert!(validate::aws_region("us-east-1"));
+  assert!(validate::aws_region("us-east-2"));
+  assert!(validate::aws_region("us-west-1"));
+  assert!(validate::aws_region("eu-west-1"));
+}
+
+#[test]
+fn invalid_aws_region()
+{
+  assert_eq!(validate::aws_region("us-east-1a"), false);
+  assert_eq!(validate::aws_region("us-east-1 "), false);
```

Comment:
Hopefully these were the kind of tests you were thinking of in https://github.com/mozilla/fxa-email-service/pull/14#discussion_r185982806 @rfk. Splitting out the separate `validate` module made writing them super-easy.